### PR TITLE
Review

### PR DESCRIPTION
**Pull request by s6nic**
Modified Review branch

### CODE REVIEW

**Comment by confidencedenku** [comment]
I checked everything and it looks great, but can you modify line 2?

**Review by confidencedenku — CHANGES_REQUESTED** [review]
Looks great, but could you modify line two of the code?

**Review by hinaahmed26 — APPROVED** [review]
keep it up , good job.

File: file2.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+hello world
```
Comment (s6charlsee) [inline file2.txt:1]: nic please change hello world to hello africa

**Review by s6charlsee — CHANGES_REQUESTED** [review]

**Review by colagene — APPROVED** [review]
Everything looks great, good job until the customer complained.

**Review by confidencedenku — APPROVED** [review]
Good work done, Nic. I love it.

**Review by s6charlsee — APPROVED** [review]
nic you are good to go,,Good work

**Pull request merged**